pyproject.toml

```diff
@@ -32,7 +32,7 @@ classifiers = [
     "Typing :: Typed",
 ]
 
-dependencies = ["fsspec>=2023.6.0", "lakefs-client>=0.105.0"]
+dependencies = ["fsspec>=2023.6.0", "lakefs-sdk>=0.113.0.2"]
```

Comment:
Let's pin this to 1.0 as a lower bound.

Reply:
Done in [f7f349f](https://github.com/appliedAI-Initiative/lakefs-spec/pull/107/commits/f7f349fa5dc88ae82f3eb096cea25e062b2b67f4).